interface-elements-for-desktop/articles/rich-text-editor/track-changes/enable-track-changes.md

```diff
@@ -0,0 +1,59 @@
+# Enable Track Changes
+
+The Rich Text Editor allows you to use Track Changes to make revisions in a document.
+
+## Turn On Track Changes
+
+On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, click the **Track Changes** button.
+
+![track-changes-enable](../../../images/rich-text-editor-track-changes-enable.png)
+
+When Track Changes is turned on, changes are indicated by red lines in the margin.
+
+![IMAGE](../../../images/rich-text-editor-track-changes-simple-markup.png)
+
+When it's turned off, the Rich Text Editor stops marking changes, but the lines are still displayed in the document.
+
+## View Changes
+
+Use the **Display for Review** drop-down list to specify how to display changes in the document. On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, open the **Display for Review** list. s
+
+![IMAGE](../../../images/rich-text-editor-track-changes-display-for-review.png)
+
+Select one of the following modes:
+
+* **All Markup** - displays revisions in details.
+    ![IMAGE](../../../images/rich-text-editor-track-changes-all-markup.png)
+
+* **Simple Markup** - displays revisions indicated by a red line in the margin.
+    ![IMAGE](../../../images/rich-text-editor-track-changes-simple-markup.png)
+
+* **No Markup** - displays the document without any visible revisions (as if all revisions are accepted).
```

Comment:
"any visible revisions" Why 'visible'?

Reply:
Because changes are in the document, but they are not visible (there are no markup)

Reply:
Ok